lib/puppet/face/node/clean.rb

```diff
@@ -47,6 +47,14 @@ def cleanup(node)
   end
 
   class LoggerIO
+    def debug(message)
+      Puppet.debug(message)
+    end
+
+    def warn(message)
+      Puppet.warning(message) unless message =~ /cadir is currently configured to be inside/
```

Comment:
I know we decided that we don't want to issue this warning for puppet6, but I assumed that was being handled elsewhere? Why do we need to account for that particular message here?